sql/update/llm_db_update.sql

```diff
@@ -0,0 +1,37 @@
+-- llm_db_update.sql ---
```

Comment:
:thinking: can we add all the create tables to this file and then find a way to create these tables any time the file changes? Note: this comment is a different approach to https://github.com/genenetwork/genenetwork3/pull/165/files#r1638117062 and I'll let you decide which may work better.

Reply:
This makes sense. Aggregating all tables for GNQA will be added to my to-do list for today.